h2o-core/src/test/java/water/fvec/TestFrameBuilder.java

```diff
@@ -0,0 +1,245 @@
+package water.fvec;
+
+import water.DKV;
+import water.Key;
+
+import java.util.HashMap;
+
+/**
+ * Class used for creating simple testing frames using builder pattern
+ * <p>
+ * Example usage:
+ * {@code
+ * final Frame builder = new TestFrameBuilder()
+ * .withName("testFrame")
+ * .withColNames("ColA", "ColB")
+ * .withVecTypes(Vec.T_NUM, Vec.T_STR)
+ * .withDataForCol(0, ard(Double.NaN, 1, 2, 3, 4, 5.6, 7))
+ * .withDataForCol(1, ar("A", "B", "C", "E", "F", "I", "J"))
+ * .withChunkLayout(2, 2, 2, 1)
+ * .build();
+ * }
```

Comment:
Seems like latest reformatting broke the code example :)
Also you want to embed the code block in <pre> </pre> tags, or otherwise it won't look good in the actual doc snippets.